stacks/ecs.yml

```diff
@@ -463,7 +462,7 @@ Resources:
               command: "(crontab -uroot -l 2> /dev/null; echo '* * * * * /root/cwdiskusage.sh') | crontab -u root -"
             03_fstrim_reclaim_disk:
               # http://docs.aws.amazon.com/AmazonECS/latest/developerguide/CannotCreateContainerError.html
-              command: "(crontab -uroot -l 2> /dev/null; echo '*/5 * * * * docker ps -qa | xargs docker inspect --format=\"{{ .State.Pid }}\" | xargs -IZ /sbin/fstrim /proc/Z/root/') | crontab -u root -"
+              command: '(crontab -uroot -l 2> /dev/null; echo ''*/5 * * * * docker ps -qa | xargs docker inspect --format="{{ .State.Pid }}" | xargs -IZ /sbin/fstrim /proc/Z/root/'') | crontab -u root -'
```

Comment:
Oh hey, a situation where prettier prefers single quotes?

Reply:
Yeah, not sure why escaped single quotes is better than escaped double, but apparently it is.